### DISCUSSION TITLE
请问有没有中文版呀？

### DISCUSSION THREAD

**Discussion by pluveto** [the1812/Bilibili-Evolved#2390] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/50045289/136885999-d3dbfa71-4bf2-4ba7-8a1b-397765e66bbb.png)
现在只能选择英文和日语

**Comment by Geeyun-JY3** [comment, marked as answer]
这是个功能，把它关掉就是了……

看 #1301 最后一条
